10.hello-docker-compose/docker-compose.yml

```diff
@@ -60,8 +62,8 @@ services:
   redis:
     image: "redis:alpine"
     ports:
-      - "6379:6379"
+      - "6379:6380"
```

Comment:
Why is the default port changed here?

Reply:
Changed port of redis to 6380 as 6379 would mostly likely be already in use because of the redis container that dapr spins up during "dapr init"

Reply:
I can’t test this change currently / but if the change is to avoid port conflict would you not change the HOST port rather than the CONTAINER port?

Reply:
You are right, my bad. Swapped them and verified changes.